de-lib/src/main/java/org/iplantc/de/fileViewers/client/views/PathListViewer.java

```diff
@@ -95,19 +93,20 @@ protected void onDragDrop(DndDropEvent e) {
                                                                      5000));
             }
 
+
         }
 
         @Override
         protected void onDragEnter(DndDragEnterEvent event) {
-            handleDropStatus(event.getDragSource().getData(),
+         /*   handleDropStatus(,
```

Comment:
Probably an oversight. I started working on this issue 2 and trying out several approach when debugging this issue but then 2 weeks back before i had to stop work on this and start work on IT summit presentation.